primitives/consensus/babe/src/lib.rs

```diff
@@ -379,11 +404,11 @@ sp_api::decl_runtime_apis! {
 		fn current_epoch_start() -> Slot;
 
 		/// Returns information regarding the current epoch.
-		fn current_epoch() -> Epoch;
+		fn current_epoch() -> StandaloneEpoch;
```

Comment:
This is a breaking API change that would require proper versioning in the runtime (and handling later on when using it). I think it would be preferable to just add `current_config` and `next_config` methods.

Reply:
Actually, scrape this one. I think just returning everything in a single method like you do here makes more sense. Let's move this type `https://github.com/paritytech/substrate/blob/master/client/consensus/babe/src/lib.rs#L134-L149` to primitives and replace the existing `Epoch` one.